build/args/release.gn

```diff
@@ -0,0 +1,7 @@
+import("all.gn")
+is_component_build = false
+is_official_build = true
+enable_widevine = true
+proprietary_codecs = true
+is_cfi = false # TODO(nornagon): why disable this?
+ffmpeg_branding = "Chrome"
```

Comment:
We can move `proprietary_codecs`, `enable_widevine` and `ffmpeg_branding` to `all.gn` and also need to add `is_component_ffmpeg = true`